src/components/pages/AboutPage/TeamMember.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '../../../utils/fontAwesome';
+import '../../../css/pages/AboutPage.css';
+
+class TeamMember extends React.Component {
+  static propTypes = {
+    name: PropTypes.string.isRequired,
+    title: PropTypes.string.isRequired,
+    favMovie: PropTypes.string.isRequired,
+    favShow: PropTypes.string.isRequired,
+    favGenre: PropTypes.string.isRequired,
```

Comment:
To make this more customizable/extensible, I think it might be better to pass in the favorites/extra info just as like an array or object or something, so you can adjust on the fly which info each person has, and it doesn't need to all be the same. You could also potentially just not have info on people this way if you wanted.

So potentially something like this:
```suggestion
  static propTypes = {
    name: PropTypes.string.isRequired,
    title: PropTypes.string.isRequired,
    imgUrl: PropTypes.string.isRequired,
    favorites: PropTypes.arrayOf(PropTypes.shape({
      type: PropTypes.string, 
      value: PropTypes.string,
    })),
  }
  
  static defaultProps = {
    favorites = [],
  }
```